src/createDijit.ts

```diff
@@ -166,7 +165,7 @@ function resolveCtor<D extends DijitWidget>(Ctor: DijitWidgetConstructor<D> | st
 				handle.remove();
 				reject(error);
 			});
-			const mid = Ctor;
+			const mid = <string> Ctor;
```

Comment:
As mentioned in other channels `node_modules` is cached on Travis.  So once you have a branch that has cached, you would have to clear it to get the same behaviour.